bin/main.rb

```diff
@@ -1,89 +1,46 @@
 #!/usr/bin/env ruby
-# rubocop:disable Metrics/PerceivedComplexity,Metrics/CyclomaticComplexity
 
-class Users
-  MAX_COUNTS = 9
-  SUCCESS_OUTCOMES = [
-    [1, 2, 3], [4, 5, 6], [7, 8, 9], [1, 4, 7],
-    [2, 5, 8], [3, 6, 9], [1, 5, 9], [3, 5, 7]
-  ].freeze
+require './lib/users'
+require './lib/logic'
```

Comment:
Use **require_relative** for files within the folder structure. **Require** should be used for external files, like gems, while **require_relative** should be used for referring to files within your directory.
In addition to that, check the **path** here, it should be **'../lib/users**' and '**../lib/logic**'.